src/cargo/ops/cargo_remove.rs

```diff
@@ -14,7 +14,7 @@ pub struct RemoveOptions<'a> {
     /// Package to remove dependencies from
     pub spec: &'a Package,
     /// Dependencies to remove
-    pub dependencies: Vec<String>,
+    pub dependencies: &'a [String],
```

Comment:
How come this was changed?  I thought we were intentionally sticking to owned types?

Reply:
Holdover from previous design. Fixed.